cp-ksql-server/include/etc/confluent/docker/admin.properties.template

```diff
@@ -0,0 +1,4 @@
+{% set security_props = env_to_props('KSQL_', '') -%}
```

Comment:
This file seems to be identical to `ksql-server.properties.template`. It also doesn't seem to be used anywhere in the image. Maybe we can drop it?